src/plugins/intel_gpu/tests/functional/shared_tests_instances/single_layer_tests/slice.cpp

```diff
@@ -0,0 +1,78 @@
+        Slice8SpecificParams{ {{{}, {{ 2, 2, 2, 2 }}}}, { 0, 0, 0, 0 }, { 2, 2, 2, 2 }, { 1, 1, 1, 1 }, {} },
+        Slice8SpecificParams{ {{{}, {{ 2, 2, 2, 2 }}}}, { 1, 1, 1, 1 }, { 2, 2, 2, 2 }, { 1, 1, 1, 1 }, {} },
+        Slice8SpecificParams{ {{{}, {{ 2, 2, 4, 3 }}}}, { 0, 0, 0, 0 }, { 2, 2, 4, 3 }, { 1, 1, 2, 1 }, { -4, 1, -2, 3 } },
+        Slice8SpecificParams{ {{{}, {{ 2, 2, 4, 2 }}}}, { 1, 0, 0, 1 }, { 2, 2, 4, 2 }, { 1, 1, 2, 1 }, {} },
+        Slice8SpecificParams{ {{{}, {{ 1, 2, 4, 2 }}}}, { 0, 1, 0, 1 }, { 10, 2, 4, 2 }, { 1, 1, 2, 1 }, {} },
+        Slice8SpecificParams{ {{{}, {{ 1, 2, 4, 2 }}}}, { 1, 0, 1, 0 }, { 2, 4, 2, 10 }, { 1, 2, 1, 1 }, { -1, -2, -3, -4 } },
+        Slice8SpecificParams{ {{{}, {{ 10, 2, 4, 2 }}}}, { 9, 1, 3, 0 }, { 0, 0, 0, 1 }, { -1, -1, -1, 1 }, {} },
+        Slice8SpecificParams{ {{{}, {{ 10, 2, 4, 2 }}}}, { 19, 1, -1, 0 }, { -10, 0, 0, -1 }, { -1, -1, -1, 1 }, {} },
+        Slice8SpecificParams{ {{{}, {{ 3, 2, 4, 200 }}}}, { 0, 1, -1, -1 }, { 3, 2, 0, 0 }, { 1, 1, -2, -1 }, {} },
+        Slice8SpecificParams{ {{{}, {{ 2, 4, 5, 5, 68 }}}}, { 0, 1, 0, 0, 0 }, {
+                std::numeric_limits<std::int64_t>::max(),
+                std::numeric_limits<std::int64_t>::max(),
+                std::numeric_limits<std::int64_t>::max(),
+                std::numeric_limits<std::int64_t>::max(),
+                std::numeric_limits<std::int64_t>::max() }, { 1, 1, 1, 1, 16 }, {} },
+        Slice8SpecificParams{ {{{}, {{ 10, 12 }}}}, { -1, 1 }, { -9999, 10 }, { -1, 1 }, {} },
+        Slice8SpecificParams{ {{{}, {{ 5, 5, 5, 5 }}}}, { -1, 0, -1, 0 }, { -50, -1, -60, -1 }, { -1, 1, -1, 1 }, {} },
+        Slice8SpecificParams{ {{{}, {{ 1, 5, 32, 32 }}}}, { 0, 2, 5, 4 }, { 1, 4, 28, 27 }, { 1, 1, 1, 1 }, { 0, 1, 2, 3 } },
+        Slice8SpecificParams{ {{{}, {{ 1, 5, 32, 20 }}}}, { 0, 1, 0, 0 }, { 1, 3, 32, 20 }, { 1, 1, 1, 1 }, { 0, 1, 2, 3 } },
+        Slice8SpecificParams{ {{{}, {{ 2, 5, 32, 20 }}}}, { 0, 0, 10, 0 }, { 1, 3, 20, 20 }, { 1, 1, 1, 1 }, { 0, 1, 2, 3 } },
+        Slice8SpecificParams{ {{{}, {{ 1, 5, 32, 32 }}}}, { 0, 0, 20, 20 }, { 1, 5, 25, 26 }, { 1, 1, 1, 2 }, { 0, 1, 2, 3 } },
+        Slice8SpecificParams{ {{{}, {{ 2, 5, 32, 32 }}}}, { 0, 0, 0, 20 }, { 1, 2, 30, 30 }, { 1, 1, 2, 1 }, { 0, 1, 2, 3 } },
+        Slice8SpecificParams{ {{{}, {{ 1, 5, 32, 20 }}}}, { 0, 0, 2, 10 }, { 1, 3, 32, 20 }, { 1, 1, 1, 1 }, { 0, 1, 2, 3 } },
+        Slice8SpecificParams{ {{{}, {{ 2, 5, 32, 32 }}}}, { 0, 1, 0, 10 }, { 1, 5, 32, 30 }, { 1, 1, 1, 1 }, { 0, 1, 2, 3 } },
+        Slice8SpecificParams{ {{{}, {{ 1, 5, 32, 20 }}}}, { 0, 1, 2, 10 }, { 1, 5, 32, 18 }, { 1, 1, 1, 2 }, { 0, 1, 2, 3 } },
+        Slice8SpecificParams{ {{{}, {{ 2, 8, 32, 20 }}}}, { 0, 0, 2, 10 }, { 1, 8, 32, 18 }, { 1, 2, 1, 2 }, { 0, 1, 2, 3 } },
+        Slice8SpecificParams{ {{{}, {{ 2, 8, 32, 20 }}}}, { 0, -20, -15 }, { 2, -5, 3 }, { 1, 1, 1 }, { 0, 2, 1 } }
```

Comment:
Please ensure that all of those test cases are meaningful and don't just duplicate each other in terms of tested functionality. Maybe some of those can be moved to nightly scope or removed at all. 

Note: test cases with `*smoke*` prefix are executed on pre-commit stage and we don't want precommit time to increase significantly (it's already too slow). So need to be careful when adding new tests into smoke scope

Reply:
Basically, imo we should prepare such data only once(ideally even in stochastic way) - and run the operators on all supported devices with the same data. Of course, there should be also possibility to added specific data for corner cases, but the point is that it should be created only once for all platforms supported. Simply - refactoring of those tests is needed, which is out of the scope of this task.

To be honest, I have just follow a convention that such data(and file) is prepared for each device separately. I have asked other people and they confirmed  me that this is what they do: they just copy the data in many cases :) 
Originally, I just wanted to modify the functional tests on cpu to run it on GPU as well, but I was asserted that it is not the convention here. 

About duplicating - generally speaking, data tests prepared for cpu device are good enough already IMO - they already test all the edge cases and it is hard to not duplicate them in that sense - GPU has to pass them.  The only thing that comes to my mind is that those inputs shapes should be generated randomly, but it would require refactoring, which is out of the scope of this task.

In other words, tests needs refactoring, which is out of the scope of this task, so I just limited myself to add comparison tests for now.

Reply:
> Basically, imo we should prepare such data only once(ideally even in stochastic way) - and run the operators on all supported devices with the same data. 

Such approach is used in conformance tests. For functional tests there are no strict rules on test scope or input parameters - that is controlled per plugin with some reuse of common things.

 > I have asked other people and they confirmed me that this is what they do: they just copy the data in many cases

Each device/plugin may have specific cases which may impact the scope. For instance, GPU plugin needs f16 precision tests while CPU - doesn't. So there's nothing wrong with customization of functional tests scope for each plugin w.r.t the actual implementation/supported features/etc. 

> About duplicating - generally speaking, data tests prepared for cpu device are good enough already IMO - they already test all the edge cases and it is hard to not duplicate them in that sense - GPU has to pass them. The only thing that comes to my mind is that those inputs shapes should be generated randomly, but it would require refactoring, which is out of the scope of this task.

The thing is that model compilation for GPU is much slower than for CPU, and we can't introduce hundreds or even thousands of tests cases for each operation as testing time growth too fast. So if we can split test cases and keep only critical things for precommit scope and remove / move to nightly the rest, then ideally it should be done. I'd suggest checking exec time for current instances, and if it fits into 1-2m limit, then OK, we can keep it as is.